storages/backends/ftp.py

```diff
@@ -41,13 +41,14 @@ class FTPStorageException(Exception):
 class FTPStorage(Storage):
     """FTP Storage class for Django pluggable storage system."""
 
-    def __init__(self, location=None, base_url=None):
+    def __init__(self, location=None, base_url=None, encoding=None):
         location = location or setting('FTP_STORAGE_LOCATION')
         if location is None:
             raise ImproperlyConfigured("You must set a location at "
                                        "instanciation or at "
                                        " settings.FTP_STORAGE_LOCATION'.")
         self.location = location
+        self.encoding = encoding or settings('FTP_STORAGE_ENCODING') or 'latin-1'
```

Comment:
This should be `setting(` instead of `settings(`.

Reply:
Thanks. Fixed